src/browser_action.js

```diff
@@ -421,17 +421,22 @@ browseraction.showEventsFromFeed_ = function(events) {
   // multi-day events (i.e., started last week, ends next week) will be shown
   // under today's date header, not under the date it started.
   var headerDate = moment().hours(0).minutes(0).seconds(0).millisecond(0);
-  $('<div>')
-      .addClass('date-header')
-      .text(headerDate.format('dddd, MMMM D'))
-      .appendTo(calendarEventsDiv);
+
+  // Insert a new div for every day, that contains all events of a single day (necessary for
+  // 'position: sticky')
+  var calendarDay =
+      $('<div>', {class: 'calendar-day'})
```

Comment:
Any particular reason this doesn’t use the common pattern of `$('<div>').addClass(…);` ?

Reply:
I was in _inspired_ by `var calendarEventsDiv = $('<div>', {id: 'calendar-events'});` and re-used it for classes. Will change it.